images/gh-gl-sync/SpackCIBridge.py

```diff
@@ -536,14 +567,21 @@ def sync(self):
             self.setup_git_repo()
 
             if self.main_branch:
-                # Find the currently running main branch pipeline, if any, and get the sha
+                # Find the currently running main branch pipeline, if any, and get the sha.
+                # Also get the latest commit on the main branch that has a completed pi
                 main_branch_pipelines = self.get_pipelines_for_branch(self.main_branch, self.time_threshold_long)
                 for sha, pipeline in main_branch_pipelines.items():
+                    if self.latest_tested_main_commit is None and \
```

Comment:
Looks like 36 hrs is the long time threshold.  If that's what we wanted when looking for the currently running pipeline, is it still enough, now that we're looking for the latest completed one?

Reply:
I don't know one way or another yet, just thinking out loud.

Reply:
36 hours is *probably* safe to get the most recent completed develop pipeline, but I don't see any harm in doubling that either, so I'll do so now.